semantic_conventions/trace/general.yaml

```
@@ -94,4 +94,30 @@ groups:
           or application security context. The value would come from the scope associated
           with an [OAuth 2.0 Access Token](https://tools.ietf.org/html/rfc6749#section-3.3)
           or an attribute value in a [SAML 2.0 Assertion](http://docs.oasis-open.org/security/saml/Post2.0/sstc-saml-tech-overview-2.0.html).
-        examples: 'read:message, write:files'
\ No newline at end of file
+        examples: 'read:message, write:files'
+  - id: code
+    prefix: code
+    brief: >
+      These attributes allow to report this unit of code and therefore to provide more context about the span.
+    attributes:
+      - id: function
+        type: string
+        brief: >
+          The method or function name, or equivalent (usually rightmost part of the code unit's name).
+        examples: serverRequest
```

```suggestion
        examples: serveRequest
```

I think _serving a request_ makes more sense for a class called `MyHttpService` rather than having a "server request".